po/es.po

```diff
@@ -81,158 +80,150 @@ msgstr "_Cancelar"
 #: src/accountdialog.ui:22 src/remotedesktopdialog.ui:28
 #: src/screencastdialog.ui:28 src/screenshotdialog.ui:38
 msgid "_Share"
-msgstr "_Compartir"
+msgstr "Co_mpartir"
 
 #: src/accountdialog.ui:148
 msgid "Make changes before sharing the information"
-msgstr ""
+msgstr "Realizar cambios antes de compartir la información"
 
 #: src/appchooserdialog.c:154
 msgid "Failed to start Software"
-msgstr ""
+msgstr "No se pudo iniciar Software"
 
 #: src/appchooserdialog.c:373
 #, c-format
 msgid ""
 "Select an application to open “%s”. More applications are available in <a "
 "href='software'>Software.</a>"
 msgstr ""
+"Seleccione una aplicación para abrir «%s». Hay más aplicaciones "
+"disponibles en <a href='software'>Software</a>."
 
 #: src/appchooserdialog.c:378
 msgid ""
 "Select an application. More applications are available in <a "
 "href='software'>Software.</a>"
 msgstr ""
+"Seleccione una aplicación. Hay más aplicaciones disponibles en <a "
+"href='software'>Software</a>."
 
 #: src/appchooserdialog.c:391
 #, c-format
 msgid "Unable to find an application that is able to open “%s”."
-msgstr ""
+msgstr "No se encontró ninguna aplicación capaz de abrir «%s»."
 
 #: src/appchooserdialog.c:396
 msgid "Unable to find a suitable application."
-msgstr ""
+msgstr "No se encontró ninguna aplicación adecuada."
 
 #: src/appchooserdialog.ui:13
 msgid "Open With"
-msgstr "Abrir Con"
+msgstr "Abrir con"
 
 #: src/appchooserdialog.ui:166
-#, fuzzy
 msgid "No Applications Found"
-msgstr "Seleccionar aplicación"
+msgstr "No se encontró ninguna aplicación"
 
 #: src/appchooserdialog.ui:182
 msgid "Find Compatible Applications in Software"
-msgstr ""
+msgstr "Encuentre aplicaciones compatibles en Software"
 
 #: src/filechooser.c:363
 msgid "_Open"
 msgstr "_Abrir"
 
 #: src/remotedesktopdialog.c:190
 msgid "Pointer"
-msgstr ""
+msgstr "Puntero"
 
 #: src/remotedesktopdialog.c:193
 msgid "Keyboard"
-msgstr ""
+msgstr "Teclado"
 
 #: src/remotedesktopdialog.c:196
 msgid "Touch screen"
-msgstr ""
+msgstr "Pantalla táctil"
 
 #: src/remotedesktopdialog.c:302
 #, c-format
 msgid "Select what to share with %s"
-msgstr ""
+msgstr "Seleccione qué compartir con %s"
 
 #: src/remotedesktopdialog.c:307
-#, fuzzy
 msgid "Select what to share with the requesting application"
-msgstr "¿ Compartir esta captura de pantalla con la aplicación de destino ?"
+msgstr "Seleccione qué compartir con la aplicación solicitante"
 
 #: src/remotedesktopdialog.ui:12
 msgid "Remote desktop"
-msgstr ""
+msgstr "Escritorio remoto"
 
 #: src/screencastdialog.ui:12
-#, fuzzy
 msgid "Screen casting"
-msgstr "Captura de Pantalla"
+msgstr "Transmisión de pantalla"
```

Comment:
"Transmisión" sounds a bit awkward to me here, never seen something like that before. What about "Captura de Vídeo"?

Reply:
Sure!